wren-ai-service/src/web/development.py

```diff
@@ -1,20 +1,128 @@
 import asyncio
+import logging
+import os
+import random
 import time
+import uuid
 
-from fastapi import APIRouter
+import orjson
+from fastapi import APIRouter, BackgroundTasks
+from redislite import StrictRedis
 
+from src.utils import async_timer
+from src.web.v1.services.ask import (
+    AskRequest,
+    AskResponse,
+    AskResultRequest,
+    AskResultResponse,
+)
+
+logger = logging.getLogger("wren-ai-service")
 router = APIRouter()
 
+REDIS_DB = (
+    StrictRedis(
+        host=os.getenv("REDIS_HOST", "redis"),
+        port=int(os.getenv("REDIS_PORT", 6379)),
+    )
+    if int(os.getenv("WORKERS", 1)) > 1
+    else StrictRedis(
+        "./redis.db",
+    )
+)
+
+
+@async_timer
+async def dummy_ask_task(ask_request: AskRequest):
+    """
+    settings:
+    uvicorn workers 4
+
+    {
+        "user_class_name": "DummyAskUser",
+        "fixed_count": 100
+    }
+
+    users = 100
+    spawn-rate = 50
+    run-time = 20s
+
+    with only await asyncio.sleep, number of finished task is about 300
+    with time.sleep(0.5) added, number of finished task is about 100
+    with time.sleep(1) added, number of finished task is about 4
+    with await asyncio.to_thread(time.sleep, 0.5) added, number of finished task is about 230
+    with await asyncio.to_thread(time.sleep, 1) added, number of finished task is about 240-250
+    """
+    await asyncio.sleep(random.randint(3, 7))
+
+    # SYNC_SLEEP_TIME = 0.5
+    # time.sleep(SYNC_SLEEP_TIME)
+    # await asyncio.to_thread(time.sleep, SYNC_SLEEP_TIME)
+
+    REDIS_DB.hset(
+        "test_ask_results",
+        ask_request.query_id,
+        AskResultResponse(
+            status="finished",
+        ).model_dump_json(),
+    )
+
+
+def get_dummy_ask_task_result(
```

Comment:
it seems only be used in get_dummy_ask_result, I think we can merge them into the same one.